README.md

```diff
@@ -0,0 +1,38 @@
+# Leaderboard-Setup-Project
+ This is a Leaderboard API app that stores users score
+
+# Additional description about the project and its features.
+
+# Built With
+  1. HTML
+  2. CSS, Javascript
+  3. Linters (Webhint, Stylelint, Lighthouse).
+  4. Webpack Module Bundler.
+
+# Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+Setup: Clone https://github.com/oluyaratosin123/Leaderboard-Setup-Project.git into the destination folder of your local machine.
+Install: Install linters with the npx commands given in the linters.yml file.
+Usage: N/A
+Run tests: Run syntax tests with the use of the npx commands given in the linters.yml file.
+Deployment: Run index.html on your local server.
+
+# Live Version
+Live Demo Link: 
+# Authors
+
+GitHub: @oluyaratosin123
+LinkedIn: Oluyara Edward
```

Comment:
- [ ] [OPTIONAL]: To keep your repo even more professional, you should consider adding the actual links to your social media.